api/src/main/java/jakarta/data/spi/provider/DataProvider.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+package jakarta.data.spi.provider;
+
+import java.lang.annotation.Annotation;
+import java.util.Set;
+
+import jakarta.data.exceptions.MappingException;
+
+/**
+ * <p>Jakarta Data providers can make themselves discoverable by
+ * Jakarta EE products via the Jakarta Data Provider SPI. This enables
+ * Jakarta EE products with CDI support to provide CDI dependency injection
```

Comment:
That point is not true; once the CDI extension is part of the spec, you can do it without it:

https://jakarta.ee/specifications/cdi/4.0/jakarta-cdi-spec-4.0.html#spi_full

Reply:
Thanks for pointing this out.  I can see where this statement can be misunderstood. I've corrected it in commit https://github.com/jakartaee/data/pull/130/commits/3b3e7260e7eb69230b84027297d820c123585d45